helpers.py

```diff
@@ -254,8 +259,10 @@ def get_sentences_speaker_mapping(word_speaker_mapping, spk_ts):
 def get_speaker_aware_transcript(sentences_speaker_mapping, f):
     for sentence_dict in sentences_speaker_mapping:
         sp = sentence_dict["speaker"]
+        st = sentence_dict["start_time"]
+        et = sentence_dict["end_time"]
         text = sentence_dict["text"]
-        f.write(f"\n\n{sp}: {text}")
+        f.write(f"\n\n{sp} [{st} - {et}]: {text}")
```

Comment:
I don't think this is needed since the timings are already in the SRT, so the TXT doesn't need to contain any timing

Reply:
Ok, I have removed st and et